features/camel/src/main/feature/feature.xml

```diff
@@ -235,7 +235,9 @@
     </feature>
 
   <feature name="camel-quartz2" version="${camel.version}" start-level="50">
-        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.c3p0/${c3p0-bundle-version}</bundle>
+      <!-- Bundle wrap & deploy mchange's version of c3p0 0.9.5.3 to force quartz2's transitive
+    dependency on 0.9.5.2 to use the newer version instead -->
+        <bundle dependency="true">wrap:mvn:com.mchange/c3p0/${c3p0-bundle-version}</bundle>
```

Comment:
since we need to do wrap can you fill out all of the fields so it has a good version and symbolic name see here for an example https://github.com/codice/ddf/blob/master/catalog/spatial/spatial-app/src/main/resources/features.xml#L167

Reply:
This is good feedback and should be fixed, but it might be OBE if Camel's quartz2 component is removed entirely. That decision (to fix or remove) is being pushed to a different PR. This one is merely for getting us off the dependency.